packages/underscore-redirects/src/print.ts

```diff
@@ -0,0 +1,26 @@
+import type { RedirectDefinition } from './redirects';
+
+export function print(
```

Comment:
We could have a comment that shows how's the printed output. Easier to "debug" for us humans :)